python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1097,6 +1128,56 @@ def do_evaluate(
             columns = broadcast(*columns)
         return DataFrame(columns)
 
+    if not POLARS_VERSION_LT_128:  # pragma: no cover
```

Comment:
Can this `if` statement be included in the `if` statement in the `evaluate` body instead?

Reply:
Yes, done in https://github.com/rapidsai/cudf/pull/18463/commits/749bb3e4696a084a2a0d749dcab82eea1d4567ec